RockPaperScissors/RockPaperScissors.cs

```diff
@@ -4,22 +4,91 @@ namespace RockPaperScissors
 {
     class Program
     {
+        public static int pScore = 0;
+        public static int cScore = 0;
+
         public static void Main()
         {
-            Console.WriteLine("Enter hand 1:");
-            string hand1 = Console.ReadLine().ToLower();
-            Console.WriteLine("Enter hand 2:");
-            string hand2 = Console.ReadLine().ToLower();
-            Console.WriteLine(CompareHands(hand1, hand2));
+            //the user is allowed a choice of hand to play with
+            Console.WriteLine("Do you choose Rock, Paper, or Scissors?");
+            string pHand = Console.ReadLine().ToLower();
+            string cHand = System.String.Empty;
+
+//generate random number and assign to an integer
+            Random rng = new Random();
+            int ran = rng.Next(0, 2);
+
+//assign random selected number to rock, paper, or scissors and call that the computer's hand
+            if (ran == 0)   
+            {
+                cHand = "rock";
+            }else if (ran == 1)
+            {
+                cHand = "paper";
+            }else
+            {
+                cHand = "scissors";
+            }
+
+//lets you know the computer choice
+//outputs the winning results
+            Console.WriteLine("Computer has chosen "+cHand); 
+            CompareHands(pHand, cHand);
+            Console.WriteLine("You: " +pScore);
+            Console.WriteLine("Me: " +cScore);
 
             // leave this command at the end so your program does not close automatically
             Console.ReadLine();
+            Replay();
+            
         }
         
-        public static string CompareHands(string hand1, string hand2)
+
+        public static void CompareHands(string pHand, string cHand)
```

Comment:
some comments on this method header would be nice.
Something like: this methods increments the pScore by 1 if pHand is the winning hand or increments cScore by 1 if cHand is the winning hand.